run/htdocs/index.html

```diff
@@ -0,0 +1,19 @@
DHr6g8kaRrntCizvz/Z3f/d3mrZLly5dor6+noQhCSxbJUeXbXrWxPVL18U2SZYN5+qD+lbl0zuh2x4KdjF0S1lykybJ9SxX0N/fr0Z+ZXXhGhsauXrhqnjgZCvVqp4pUd9bGB/13Qe4b968yZ07dwyeWg4jR0rKZvmh3PPC1YXEJ8RLjTm275iofx+XAslyY7j1TMhGCYKMNdrkBkMb19M8KCy5KUePHuXTTz+1+noGxfHjx2lqamJE2ggKCgukxqjNGYbFQ9ngCrQqLj1Sfgo3dz4QXiD2/Kt37tzJX/zFX5hwCuPg9XrVklRNzRP3BrEu90rGIwL3/TvhtoN9DB1EgGjK3r17bW/oShCuZFOJdHOGgzv88YfOnuAHSQZezDF0EOSZd8LQKyoqaGhoIGlYEouL5TrfNDxsEIVDAA9fiH/y8DG4uu47BTsZ+h7gJ4cPH8bj8WiuZIoUXr58qcYS1n8iR3m9XXWbujt1IHjqf42gosqiDXis7SqlsQf4VWVlZVR9fT3jx4836TThQ3Hbi9cWS2sFqLlz0dzi94j7LstDeIF59z3isJOhKyy5EefPn9fMOY8Udu/ejcvlImtCFvmz86XGHNyuZhN2AX9l1rXpQAtwBijatWsXP/nJT6y+ngHh8XjUnL8WAUg12g7/Hfi14Rf2DsEuwTgIYslZRZ6RgaILt3arXO7c6/EGaqDNc8HDwU6wNvoeCqdOnaK5uZmUESksWCrHXnxw7wF3qu6AqMO3ff9ys2EnQwc/S84q0chQCG7OIFupdrH8Is+bngM8R/TStht2Ad5vvvmGhoaGkAdbAdVtX1csHRMJWs2PAa2DHPq9gN0M3VKWXCh89dVXeDweZsybwZgsuRSZGoQT+VhrtJYHxzPgvM/ns7Si7W3o6+tj926xIGvqkrpHNfR3nr5qBOxm6CpL7sCBAxZfyrehtafaq+5XnCpVlYGNJrwYCdu678eOHaOtrY3U9FRp9Z47VXd4cO8BiN5o1rCwbAa7GTr4yTNWdVx9G6qrq7l69SoxMTHSeVy1OQPcQ7RGsit2Ab7y8nKamppCHhxJKJTXko0l0loBQdH2w4iup9972NHQ94N1LLm3QVnNFxcvJnm4XA10kNuuVZo50ngIXPZ4PLZ6wfb09KjXozPa/t5t98OOhm55x9U34fV6+eqrrwD53Hnb8zYunFVbPv3GnCszFDvB2tLVN3Ho0CE6OzvJGJPBzHkzpcbcunpLyHSBC79s1nvY09DB/weyy+py9uxZHj58SNKwJJaXLJcaU7q7VDRnEC67JOfVUuwCKCsro62tzeprAQLR9pKNJdLFL0GU1/0IY38P7GvoewEUlpzVUHLnupoz2DN3PhDuAtfdbrctXrAul0stW5YtSfV6vRzbr3Yz/k5UnRkFuxq6pVpywQhuziDrtj+4/0DpROLm3donWtpeORj79u2ju7ubsePHMm3mNKkxNy7foOlpE4gAXKh20t8r2NXQbcOS279/P+3t7YwaPYrZC+XEKYIor0cQNNN3BTtApLQ6O60NVuuRcw5iIO5GWz3Bdx52NXTw5z+tZskpbvu6reukmzOU7lYfODvnzgdCtIZckAAAA/1JREFUNVDb29trKY+ho6OD0lJxD2VJMl6PlxMHTygf37vtb8DOhn4Ui1lywc0ZZEky1y9dp+FRAwj30Z5c3sFhOXlm9+7d9Pb2MmHSBGmt/MvnLtPa3AqC7nosxOHfO9ipeu1NKCy5ogMHDvCnf/qnEb+Abdu20dfXR0xsDH/2oz+TGuNP7YDY74YrzWwFdgJ/WVpaisvlIjFRsp2zgVDc9ufNz9m4aGOIowXeuO92pBq/xyD4M8BXVFTkswJLlizxIYguev4VR/52GYb7gG/Hjh0Rv+ctLS2+mJiYcO67nFLn9wx2XtFBuL6/UFhyWvtzhYO6ujrOnTsHQuFlPdqDalcNv6jIYSfwH3fs2KF2P40Udu3ahdvtBsE9+GOJIW7gZdDnd7qjyvcZNYBv27ZtEV1Z/vqv/1pZIexBz4ssFgK+oUOH+np6eiJ634uLi5X7/p8tvgfvEWH8LeD7wQ9+ENEHbvLkycoD968s/v5WwIGQUfLt3bs3Yvf82bNnPqfT6UN4UdmW3oH3iDiWAr4RI0b4+vv7I/LAXbhwQTFyFzDU2q9vGf4B8P3oRz+KyD33+Xy+X/7yl8p9t3Ol33uYhChEysR39uzZiDxwP/7xj5UH7l2hr5qB5YBv+PDhvr6+vojc96Dgp1yK4z2+c/gS8P30pz81/WFzu92+9PR05YGTE4b7bsKJUJ/xlZaWmn7fHz586HM4HD4EK1Kylc17yMLOhJlg7IHIqM4cPXqU5uZmgGa+38QLL/77HgnyzPbt2/H5fCC4E09DHP4e31EkA32A7969e6auLD/4wQ+U1fx/WvuVbYFVgG/kyJGmx0fmzZun3HeZlNp7fIdxEvD94he/MO1h6+zs9A0ZMkR54OTbfH53EY1Qr/WdPHnStPt+9+5d5Z67gXRLv/F3FO+K6w7+GnUzq9l2795Nd3c3wG1Eg73vO/rx33czS1eVnmrAKcSW6T2+x8gBfDExMb729nZTVpaSkhJlZflLi7+rnbAO8I0ePdrn8XhMue/Tp09X7vv/afF3fQ+bwDSWXENDQzBZY4LF39NOiAPaAV9FRYXh972qqkox8l5guKXf9DuMd8l1B3/Zp9Lk0Ej87ne/E22NoQKoM/wE7y568d93M4QjlUo1BNVYU7vT9/juwjSW3KxZs5SV5Y8s/o52xBbAl52d7fN6vYbe9yCq8f9h8Xf8TkNOWtM+iEIEa0b8y7/8i2EdV2/dusXmzZtBpPAyEa2K3yOABMR9T/r666+ZN2+eIZNevXqVTz75BERHlXSgy5CJ3+M7gd+iv1Y51L/vfdfNQbAN8+67fcTkv6Owez36QNiFiASbgX8xad7vArYhCDRm4CuT5n0PP/5/x3WwVy9a2hsAAAAASUVORK5CYII=" />
```

Comment:
Is this just for fun, or can the image be omitted?

Reply:
Yeah, just for fun.